src/core/ext/transport/chttp2/alpn/alpn.cc

```diff
@@ -25,7 +25,7 @@
 #include "src/core/lib/gpr/useful.h"
 
 // in order of preference
-static const char* const supported_versions[] = {"grpc-exp", "h2"};
+static const char* const supported_versions[] = {"h2"};
```

Comment:
Do we plan to have any other supported versions? Otherwise I wonder if it's worth it to just remove this complexity altogether and have a non-arr const value for `h2`

Reply:
I'm planning a follow-up PR to clean up this file (some of the APIs, e.g. `grpc_chttp2_get_alpn_version_index`, are now unused). Do you mind if we defer this to the cleanup PR?